inspirehep/modules/records/validators/__init__.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.ext
+# Copyright (C) 2017 CERN.
+#
+# Invenio is free software; you can redistribute it
+# and/or modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# Invenio is distributed in the hope that it will be
```

Comment:
What I can't understand is "from where?". There are no more Python files that have wrong licenses, there's a script in `scripts/prettify_licenses` that does this job automatically... and people still do the wrong thing. How?